tests-integration/src/cluster.rs

```diff
@@ -63,10 +65,24 @@ pub struct GreptimeDbClusterBuilder {
 }
 
 impl GreptimeDbClusterBuilder {
-    pub fn new(cluster_name: &str) -> Self {
+    pub async fn new(cluster_name: &str) -> Self {
+        let endpoints = env::var("GT_ETCD_ENDPOINTS").unwrap_or_default();
+
+        let kv_backend = if endpoints.is_empty() {
```

Comment:
1. If we should assert to use EtcdStore?
2. If we should pin the endpoints as `2379`?

Reply:
Sorry, what do you mean by "assert to use EtcdStore"? The endpoints are pvoided as env variables, I don't understand what "pin" means

Reply:
@MichaelScofield Yes. This is the current implementation. What I mean is that if we want this flexibility or just use a fixed (pinned) implementation and configs.

Reply:
The presence/absence of env `GT_ETCD_ENDPOINTS` also acts as a "switch" to indicate whether we are testing against a real Etcd. By "fix" you mean getting rid of the env var completely and test against Etcd every time even on develop's local machine, or not using its value and `let endpoints = "127.0.0.1:2379"`?

Reply:
Yep. But let's use the current solution for testing first.